Jenkinsfile

```diff
@@ -28,6 +28,10 @@ node {
     overrideTestTask: {
       for (rubyVersion in rubyVersions) {
         for(gemfile in gemfiles) {
+          if (gemfile == "rails_7" && rubyVersion == "2.6") {
+            continue
+          }
```

Comment:
Since Ruby 2.6 is end of life now shall we drop support for that (https://github.com/alphagov/gds-sso/blob/793059e97cbc6b262c15e81ae6ede4549eb55dee/gds-sso.gemspec#L17)